### DISCUSSION TITLE
Possible bug, editor removes spaces from formatted tags.

### DISCUSSION THREAD

**Discussion by paulovargatt** [GrapesJS/grapesjs#5142] [Q&A]
@artf  See the example below
[video-convert-1684981370304.webm](https://github.com/GrapesJS/grapesjs/assets/28814403/dc80eaab-d8bc-47c0-91d7-bda175d1caad)
my functions to save:
  const editorState = await this.editor.getProjectData()

  this.editor.loadProjectData(JSON.parse(pageUser.editor))

if I use this.editor.setComponents(pageUser.html);
everything seems to work fine but I lose the project data, the template is loaded however, without defined colors, background and etc.

Has anyone experienced this or know if this is a problem?

**Comment by artf** [comment]
You should load the projectData you stored before, not the html

**Reply by paulovargatt** [reply]
Yes, this is exactly what I do:
this.editor.loadProjectData(JSON.parse(projectDataJson))

everything works fine, only tags formatted with "bold", "italic" or some custom color, spaces are removed as per the video

**Reply by artf** [reply]
can you provide a reproducible live demo? From the official one I don't see such an issue
